tests/app/fixtures/openapi.json

```diff
@@ -0,0 +1 @@
s-spreadsheet":{"post":{"summary":"Process Spreadsheet","description":"Process data to generate new ads or keywords based on the template","operationId":"process_spreadsheet_process_spreadsheet_post","parameters":[{"name":"user_id","in":"query","required":true,"schema":{"type":"integer","description":"The user ID for which the data is requested","title":"User Id"},"description":"The user ID for which the data is requested"},{"name":"template_spreadsheet_id","in":"query","required":true,"schema":{"type":"string","description":"ID of the Google Sheet with the template data","title":"Template Spreadsheet Id"},"description":"ID of the Google Sheet with the template data"},{"name":"template_sheet_title","in":"query","required":true,"schema":{"type":"string","description":"The title of the sheet with the template data","title":"Template Sheet Title"},"description":"The title of the sheet with the template data"},{"name":"new_campaign_spreadsheet_id","in":"query","required":true,"schema":{"type":"string","description":"ID of the Google Sheet with the new campaign data","title":"New Campaign Spreadsheet Id"},"description":"ID of the Google Sheet with the new campaign data"},{"name":"new_campaign_sheet_title","in":"query","required":true,"schema":{"type":"string","description":"The title of the sheet with the new campaign data","title":"New Campaign Sheet Title"},"description":"The title of the sheet with the new campaign data"},{"name":"target_resource","in":"query","required":true,"schema":{"enum":["ad","keyword"],"type":"string","description":"The target resource to be updated","title":"Target Resource"},"description":"The target resource to be updated"}],"responses":{"200":{"description":"Successful Response","content":{"application/json":{"schema":{}}}},"422":{"description":"Validation Error","content":{"application/json":{"schema":{"$ref":"#/components/schemas/HTTPValidationError"}}}}}}}},"components":{"schemas":{"Body_process_data_process_data_post":{"properties":{"template_sheet_values":{"$ref":"#/components/schemas/GoogleSheetValues"},"new_campaign_sheet_values":{"$ref":"#/components/schemas/GoogleSheetValues"}},"type":"object","required":["template_sheet_values","new_campaign_sheet_values"],"title":"Body_process_data_process_data_post"},"GoogleSheetValues":{"properties":{"values":{"items":{"items":{},"type":"array"},"type":"array","title":"Values","description":"Values to be written to the Google Sheet."}},"type":"object","required":["values"],"title":"GoogleSheetValues"},"HTTPValidationError":{"properties":{"detail":{"items":{"$ref":"#/components/schemas/ValidationError"},"type":"array","title":"Detail"}},"type":"object","title":"HTTPValidationError"},"ValidationError":{"properties":{"loc":{"items":{"anyOf":[{"type":"string"},{"type":"integer"}]},"type":"array","title":"Location"},"msg":{"type":"string","title":"Message"},"type":{"type":"string","title":"Error Type"}},"type":"object","required":["loc","msg","type"],"title":"ValidationError"}}}}
```

Comment:
Why do we have this?

Reply:
For test.

Previously it was inside the test file: https://github.com/airtai/google-sheets/blob/main/tests/app/test_app.py#L165-L628